solutions/JOI 18-commuter-pass.mdx

```diff
@@ -1,13 +1,21 @@
 ---
 id: joi-18-commuter-pass
 title: JOI 2018 Commuter Pass
-author: Andi Qu
+author: Andi Qu, Nathan Wang
 ---
 
-<IncompleteSection />
+1. Use Dijkstra's to calculate distances from $s$, $u$, and $v$.
+2. Consider the dijkstra tree (keep track of all the parents of each node) from $s$ to $t$. Note that this is a DAG.
```

Comment:
Calling it a tree is kinda confusing because you say it's a DAG (also I don't think it's a tree that you're considering). Also, capitalize "dijkstra"